package.json

```diff
@@ -11,6 +11,10 @@
     "bin",
     "lib"
   ],
+  "engines" : {
+    "node" : ">=4.2.0 <5.0.0 || >=6.9.0",
+    "npm": ">=2.14.7"
```

Comment:
npm 2.14.7 is a version which comes with Node 4 LTS Argon. Node 4 version proir to 4.2 [are not maintained](https://github.com/nodejs/LTS). So I suppose this should be than
```
{
  "node": ">=4.2",
  "npm": ">=2.14.7"
}
```